### PR TITLE
Converted `configuration` docs from asciidoc to markdown

### LINKED ISSUE #21: Convert `configuration` documentation from AsciiDoc to markdown
The couple of files in https://github.com/IntellectualSites/fastasyncworldedit-documentation/tree/main/sources/configuration need to be migrated to Markdown and put in place in a new `configuration` folder in https://github.com/IntellectualSites/documentation/tree/main/fastasyncworldedit/configuration

### PR DESCRIPTION
**Pull request by specbeck**
## Overview

Fixes #21

## Description

Converts the couple files at https://github.com/IntellectualSites/fastasyncworldedit-documentation/tree/main/sources/configuration to markdown and places them in a new `configuration` folder at https://github.com/IntellectualSites/documentation/tree/main/fastasyncworldedit/configuration

### Submitter Checklist
- [x] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [x] Ensure that the pull request title represents the desired changelog entry.
- [x] New public fields and methods are annotated with `@since TODO`.
- [x] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: fastasyncworldedit/configuration/configuration.md [hunk 1]
```diff
@@ -0,0 +1,438 @@
+# Configuration
+
+Table of Contents
+
+[🏠 Return to the index](../README.md)
```
Comment (NotMyFault) [inline fastasyncworldedit/configuration/configuration.md:5]:
```suggestion
```

File: fastasyncworldedit/configuration/configuration.md [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+
+- [Introduction](#introduction)
+- [Asynchronous Operation](#asynchronous-operation)
+- [Limits Settings](#limits-settings)
+- [Queue Settings](#queue-settings)
+- [Local Queues](#local-queues)
+- [Global Queues](#global-queues)
+- [Efficiency Improvements](#efficiency-improvements)
+- [Default Configuration File](#default-configuration-file)
```
Comment (NotMyFault) [inline fastasyncworldedit/configuration/configuration.md:14]: The ToC isn't working. I suggest to remove it.

File: fastasyncworldedit/configuration/configuration.md [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+    disallowed-blocks:
+    - "minecraft:wheat"
+    - "minecraft:fire"
+    - "minecraft:redstone_wire"
+    # List of block properties that should be remapped if used in an edit. Entries should take the form
+    # "property_name[value1_old:value1_new,value2_old:value2_new]". For example:
+    #  - "waterlogged[true:false]"
+    #  - "age[7:4,6:4,5:4]"
+    #  - "extended[true:false]"
+    remap-properties: []
```
Comment (NotMyFault) [inline fastasyncworldedit/configuration/configuration.md:437]: I'd suggest removing the example config, we don't maintain this specific entry here.

File: fastasyncworldedit/configuration/permissions.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Permissions
+
+Table of Contents
+
+[🏠 Return to the index](../README.md)
+
+- [Admin permissions](#admin-permissions)
+- [User permissions](#user-permissions)
+- [Region permissions](#region-permissions)
+- [Extended WorldEdit permissions](#extended-worldedit-permissions)
```
Comment (NotMyFault) [inline fastasyncworldedit/configuration/permissions.md:10]: Same like above.

File: fastasyncworldedit/configuration/permissions.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+> [TIP] **Info:** For command related permissions, see [here](../basic-commands/main-commands-and-permissions.adoc).
```
Comment (NotMyFault) [inline fastasyncworldedit/configuration/permissions.md:12]: This isn't working with gitbook's syntax. Take a look at the other pages or the gitbook documentation how to use gitbook's custom markdown flavor. 

Additionally, the page URL is wrong. The adoc files do not exist here. Rather, use a relative URL to the command page.

File: fastasyncworldedit/configuration/permissions.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+- `fawe.admin` (allow bypassing through `/wea` command)
+- `fawe.bypass` (automatically bypass worldedit restrictions)
+- `worldedit.anyblock` (bypass worldedit's `disallowed-blocks`)
+- `worldedit.inventory.unrestricted` (bypass inventory restrictions if those are enabled)
+
+## User permissions
+
+- `fawe.limit.<limitgroup>` (Grants the user the specific group of limits.)
+- [`fawe.permpack.basic`](https://github.com/IntellectualSites/FastAsyncWorldEdit/blob/main/worldedit-bukkit/src/main/resources/plugin.yml#L31) (A bunch of worldedit perms which are okay to use on a creative server)
```
Comment (NotMyFault) [inline fastasyncworldedit/configuration/permissions.md:24]: This should be a relative URL to the corresponding gitbook page.

**Review by NotMyFault — CHANGES_REQUESTED** [review]

**Comment by NotMyFault** [comment]
You can take a look at the GH checks to preview a gitbook render of this PR.

File: fastasyncworldedit/configuration/permissions.md (hunk 6 above)
Comment (specbeck) [reply fastasyncworldedit/configuration/permissions.md:24]: I can't seem to find the required file under the same repo, is it needed to point to some other page/file?

Comment (NotMyFault) [reply fastasyncworldedit/configuration/permissions.md:15]: Wrong line I commented at, please ignore it :)

**Review by NotMyFault — APPROVED** [review]
Thanks so much :)

**Comment by specbeck** [comment]
@NotMyFault Thanks for having me as a contributor! :D

**Comment by NotMyFault** [comment]
You’re welcome :)

**Pull request merged**